Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN go mod download
 # Copy the go source
 COPY . .
 # Build
-RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o meshery-meshsync meshsync.go
+RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o meshery-meshsync main.go
```

Comment:
Thanks for this PR, @Aisuko. I’m probably confused - isn’t the name of the file to compile `meshsync.go`? https://github.com/layer5io/meshsync/tree/master/meshsync

Reply:
Yes, I am confused. Abishek pointed out https://github.com/layer5io/meshsync/blob/master/main.go to me now.

Reply:
Yep, we want to compile our file to `meshery-meshsync`.

```
usage: go build [-o output] [-i] [build flags] [packages]

```